### DISCUSSION TITLE
[Problem] The mina cli doesn't have an option to log precomputed blocks locally in the format that is sent to gcloud

### DISCUSSION THREAD

**Discussion by trevorbernard** [MinaProtocol/mina#12988] [Ideas]
It is very useful to get the state hash for the block which isn't available in the log-precomputed-blocks.

**Comment by jenr24G** [comment]
In fact, the precomputed block format doesn't have the state hash of the block that it represents in it at all. It would be useful to add a `state_hash` field to the precomputed block format
